sunbird/common_config/config.ini

```diff
@@ -1,39 +1,46 @@
 [pre-prod]
-host = # Add host Ip address followed by a slash / 
-internal_kong_ip = # Add Internal kong Ip address followed by a slash / 
-keyclockapibody = 
+host = http://localhost:3569/
+internal_kong_ip = http://localhost:3569/
+keyclockapibody = "email": "rahul@yahoo.com","password": "password"
```

Comment:
How can u expose email and password in git? 
even though its local credentials , its not a recommended practice